docs/rules/no-unreadable-iife.md

```diff
@@ -0,0 +1,36 @@
+# Forbid unreadable IIFEs
```

Comment:
I think we should standardize on the word "disallow" over "forbid". I noticed we have been inconsistent about it, but we mostly use "disallow".

Reply:
I can make a eslint rule for it.

Reply:
#1777